packages/components/react/src/link/link.variants.ts

```
@@ -0,0 +1,152 @@
+import { cva } from "class-variance-authority";
```

ESLint

[error] Unable to resolve path to module 'class-variance-authority'.

(import-x/no-unresolved)